doc/ref/floats.xml

```diff
@@ -162,7 +162,111 @@ false
 ]]></Example>
 <P/>
 
-<#Include Label="FLOAT_UNARY">
+<ManSection>
+  <Heading>Mathematical operations</Heading>
+  <Attr Name="Cos" Arg="x"/>
+  <Attr Name="Sin" Arg="x"/>
+  <Attr Name="SinCos" Arg="x"/>
+  <Attr Name="Tan" Arg="x"/>
+  <Attr Name="Sec" Arg="x"/>
+  <Attr Name="Csc" Arg="x"/>
+  <Attr Name="Cot" Arg="x"/>
+  <Attr Name="Asin" Arg="x"/>
+  <Attr Name="Acos" Arg="x"/>
+  <Attr Name="Atan" Arg="x"/>
+  <Oper Name="Atan2" Arg="y x"/>
+  <Attr Name="Cosh" Arg="x"/>
+  <Attr Name="Sinh" Arg="x"/>
+  <Attr Name="Tanh" Arg="x"/>
+  <Attr Name="Sech" Arg="x"/>
+  <Attr Name="Csch" Arg="x"/>
+  <Attr Name="Coth" Arg="x"/>
+  <Attr Name="Asinh" Arg="x"/>
+  <Attr Name="Acosh" Arg="x"/>
+  <Attr Name="Atanh" Arg="x"/>
+  <Oper Name="Log" Arg="x"/>
+  <Attr Name="Log2" Arg="x"/>
+  <Attr Name="Log10" Arg="x"/>
+  <Attr Name="Log1p" Arg="x"/>
+  <Attr Name="Exp" Arg="x"/>
+  <Attr Name="Exp2" Arg="x"/>
+  <Attr Name="Exp10" Arg="x"/>
+  <Attr Name="Expm1" Arg="x"/>
+  <Attr Name="CubeRoot" Arg="x"/>
+  <Attr Name="Square" Arg="x"/>
+  <Oper Name="Hypothenuse" Arg="x y"/>
+  <Attr Name="Ceil" Arg="x"/>
+  <Attr Name="Floor" Arg="x"/>
+  <Attr Name="Round" Arg="x"/>
+  <Attr Name="Trunc" Arg="x"/>
+  <Attr Name="Frac" Arg="x"/>
+  <Attr Name="SignFloat" Arg="x"/>
+  <Attr Name="Erf" Arg="x"/>
+  <Attr Name="Zeta" Arg="x"/>
+  <Attr Name="Gamma" Arg="x"/>
+  <Description>
+    Usual mathematical functions.
+  </Description>
+</ManSection>
+
+<ManSection>
+  <Oper Name="EqFloat" Arg="x y"/>
+  <Returns>Whether the floateans <A>x</A> and <A>y</A> are equal</Returns>
+  <Description>
+    This function compares two floating-point numbers, and returns
+    <K>true</K> if they are equal, and <K>false</K> otherwise; with the
+    exception that <K>NaN</K> is always considered to be different from
```

Comment:
`<K>` is for keywords. Shouldn't this be `<C>NaN</C>`? Of course, that is not a valid GAP input either, and even not an output (`0.0/0.0;` outputs `nan`).

Reply:
I didn't write that text. And there are tons of other issues with it. I'd rather not work on it any more than what I did so far.